src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { t } from 'i18next';
+import { HomeStackParamList } from '../navigators/HomeStack';
+import { useActiveAccount } from '../hooks/useActiveAccount';
+import { ActivityIndicatorView } from '../components/ActivityIndicatorView';
+
+type Props = NativeStackScreenProps<HomeStackParamList, 'Home'>;
+export type HomeScreenNavigation = Props['navigation'];
+
+export const HomeScreen = () => {
+  const { isLoading: loadingAccount, account } = useActiveAccount();
+
+  if (loadingAccount) {
+    return (
+      <ActivityIndicatorView
+        message={t('home-screen-loading', 'Loading account information')}
+      />
+    );
+  }
+
+  return (
+    <View style={styles.container} testID="home-screen">
+      <SafeAreaView>
+        <ScrollView
+          overScrollMode="always"
```

Comment:
Is this what we want for Android? Personally I like the overscroll behavior but I am an iOS user so I might be biased.

Reply:
Totally fair question- let's worry about this later.